### PR TITLE
Fixing timer sensor link in docs

### PR DESCRIPTION
**Pull request by DoriftoShoes**

### CODE REVIEW

File: docs/source/sensors.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ Timer sensor
 ^^^^^^^^^^^^
 
 Look at the timer sensor implementation
-`here <https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/contrib/sensors/st2_timer_sensor.py>`__.
+`here <https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/timer/base.py>`__.
```
Comment (manasdk) [inline docs/source/sensors.rst]: Timer is not a sensor any more so maybe we drop it from this doc entirely?

Comment (DoriftoShoes) [inline docs/source/sensors.rst]: You are right.  I am going to close this PR and look into adding a timer section to the rules.

Comment (Kami) [inline docs/source/sensors.rst]: I saw your Slack comment - I'm on it, but next time we should try to be more complete and also update docs when the code changes are made :P

Comment (DoriftoShoes) [inline docs/source/sensors.rst]: Totally agree.  I actually only asked you to do it because DZ said you made the code changes :P

Comment (Kami) [inline docs/source/sensors.rst]: Heh, it was lakshmi. Anyway, it doesn't matter, I just like to give Lakshmi a hard time sometimes for not writing docs :P

#961

**Pull request closed without merging**